app/scripts/views/settings/emails.js

```diff
@@ -23,6 +23,8 @@ define(function (require, exports, module) {
   const EMAIL_REFRESH_SELECTOR = 'button.settings-button.email-refresh';
   const EMAIL_REFRESH_DELAYMS = 350;
 
+  const EMAIL_ENABLED_REGEX = /.+@mozilla\.com$|.+@restmail\.net$|.+@softvisioninc\.eu$|.+@softvision\.(com|ro)$/;
```

Comment:
I'm against putting more stuff into Able at this point. I suggest using the config serializer instead: https://github.com/mozilla/fxa-content-server/blob/master/server/lib/routes/get-index.js#L22 

and make this configurable via `process.env`. @jbuck says we are aiming for content-server docker deploy in train-88, so it should get easier to just adjust `env` vars and docker deploy